src/hooks/sync-sites/use-sync-pull.ts

```diff
@@ -174,28 +169,25 @@ export function useSyncPull( {
 						cancelId: 1,
 					} );
 
-					if ( userChoice === 1 ) {
-						console.log( 'User cancelled pull operation' );
+					const isCanceledByUser = userChoice === 1;
+					if ( isCanceledByUser ) {
```

Comment:
Do we need this variable if it's used only once?

Reply:
I just wanted to avoid "magic" number `1`, since at a glance I thought that `1` means kinda `true`, as a confirmation to continue pulling, but actually it's canceling.

So to keep readability, we need either to add comment about `1` or create readable and simple variable. And the good variable name is much better than any comment.